modules/olm-channel-schema.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-packaging-format.adoc
+
+[id="olm-channel-schema_{context}"]
+= olm.channel schema
+
+The `olm.channel` schema defines a channel within a package, the bundle entries that are members of the channel, and the upgrade edges for those bundles.
+
+A bundle can included as an entry in multiple `olm.channel` blobs, but it can have only one entry per channel.
```

Comment:
```suggestion
A bundle can represent an edge in multiple `olm.channel` blobs, but it can appear only once per channel.
```

Reply:
Mashup of this suggestion + earlier peer review suggestion:
> If a bundle entry represents an edge in multiple `olm.channel` blobs, it can only appear once per channel.